src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsIssuesListener.php

```diff
@@ -94,4 +98,52 @@ protected function checkNoCodelabel($hookData, Github $github, Logger $logger, $
 			$this->addLabels($hookData, $github, $logger, $project, $labels);
 		}
 	}
+
+	/**
+	 * Checks for issue label rules
+	 *
+	 * @param   object  $hookData  Hook data payload
+	 * @param   Github  $github    Github object
+	 * @param   Logger  $logger    Logger object
+	 * @param   object  $project   Object containing project data
+	 *
+	 * @return  void
+	 *
+	 * @since   1.0
+	 */
+	protected function checkIssueLabels($hookData, Github $github, Logger $logger, $project)
+	{
+		// Set some data
+		$rfcLabel     = 'Request for Comment';
+		$addLabels    = [];
+		$removeLabels = [];
+
+		$rfcIssue    = strpos($hookData->issue->title, '[RFC]') || substr($hookData->issue->title, 0, 5) === 'RFC';
```

Comment:
`substr($hookData->issue->title, 0, 5) === 'RFC'` won't pass because you're checking if the first 5 characters of the issue title matches a 3 character string.

Reply:
hehe i have found that too but just not commited ... should be ok now.

Reply:
Actually, this could be more easily done as `strpos($hookData->issue->title, '[RFC]') !== false` since `strpos()` can return 0 which wouldn't evaluate to true in this condition.  That way you don't need the substr check and the [RFC] indicator can be anywhere in the title (as sometimes you have things like "[4.0][RFC]").

Reply:
done.